layouts/partials/html-head.hbs

```diff
@@ -18,4 +18,7 @@
     <link rel="alternate" href="/{{site.locale}}/tsc-minutes.xml" title="Node.js TSC meeting minutes" type="application/rss+xml">
     <link rel="stylesheet" href="/{{site.locale}}/styles.css" media="all">
     <link rel="stylesheet" href="/static/css/prism-tomorrow.css" media="all">
+
+    <script src="//use.typekit.net/mse5tqx.js"></script>
+    <script>try{Typekit.load();}catch(e){}</script>
```

Comment:
Hmm, wouldn't it be better to just include the font from link into css? That would make for less potential font resizing, and less javascript. :)